dist/flatpak/com.github.input-leap.input-leaps.yml

```diff
@@ -10,34 +10,63 @@
 # https://gitlab.com/freedesktop-sdk/freedesktop-sdk/-/issues/1013
 
 app-id: com.github.inputleap.InputLeap
-runtime: org.gnome.Platform
-runtime-version: '42'
-sdk: org.gnome.Sdk
-command: input-leap-flatpak
+runtime: org.kde.Platform
+runtime-version: '5.15-23.08'
+sdk: org.kde.Sdk
+command: input-leap
 
 finish-args:
     # We want to talk to the network
     - --share=network
     # Need to read our config from somewhere
     - --filesystem=xdg-config/input-leap
-    # HACK: so we can put our EIS socket somewhere accessible
-    # This won't be needed once the portal actually supports our interfaces
-    - --filesystem=xdg-run/eis-0
     # Let's show off :)
-    - --nosocket=x11
-    - --nosocket=wayland
+    - --socket=x11
+    - --socket=wayland
+    - --env=QT_QPA_PLATFORM=wayland
+    - --device=dri           # For hardware acceleration if needed
+    - --env=DISPLAY=:0       # Ensure the DISPLAY variable is set correctly
+    # - --command=input-leaps /app/bin/input-leaps
+    # - --command=input-leapc /app/bin/input-leapc
```

Comment:
should be removed

Reply:
Do you know how to export executables to /usr/bin/ from a flatpak or to even export multiple executables. I want to export /app/bin/input-leaps /app/bin/input-leapc /app/bin/input-leap